docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,143 @@
+
+Learn more about support bundles and how to configure them in our [support bundle documentation]({{< relref "../troubleshooting/support-bundles/" >}}).
+
+## Alerting
+
+### Search for alert rules from multiple data sources
+
+Allows you to search for and display alert rules for multiple data sources at the same time.
+
+### Fuzzy search on the Alert rule list view
+
+Search for namespaces or folders, evaluation groups, and alert rule names on the Alert rules list view with immediate results and regardless of typos.
+
+### Access an alert rule from a dashboard or a panel
+
+Navigate to an alert rule directly from a dashboard or a panel to easily dive into the alert rule details.
+
+{{< figure src="/media/docs/alerting/alert-rule-dashboard.png" max-width="750px" caption="Access an alert rule from a dashboard or a panel" >}}
+
+### Access a dashboard or panel from an alert rule
+
+Navigate from an alert rule straight to a dashboard or a panel associated with the alert rule to visualize your alerting data.
+
+{{< figure src="/media/docs/alerting/dashboard-alert-rule.png" max-width="750px" caption="Access a dashboard or panel from an alert rule" >}}
+
+### Preview queries for recording rules
+
+Preview queries when creating or editing recording rules, so you can visualize the results of your query before saving your recording rule.
+
+{{< figure src="/media/docs/alerting/preview-queries-recording-rule.png" max-width="750px" caption="Preview queries for recording rules" >}}
+
+### Updates alert behavior for no data evaluation
+
+Alert rules that are configured to fire when an evaluation returns no data now only fire when the entire duration of the evaluation period has finished. This means that rather than immediately firing when the alert rule condition is breached, the alert rule waits until the time set in the For field has finished and then fires, reducing alert noise and allowing for temporary data availability issues.
+
+### Improved Notification Policies view
+
+- Updates to the notification policies view make it easier to use and manage:
+- View default policy and nested policies at a glance
+- New tab for mute timings
+- View alert instances for each policy
+- View contact points and which integrations are configured for each policy
+- View inherited properties on nested policies
+- Search for label matchers and for contact points to see which notifications are going where
+
+{{< figure src="/media/docs/alerting/notification-policies-view.png" max-width="750px" caption="Improved Notification Policies view" >}}
+
+### Guidance for configuring your Alertmanager
+
+Get additional help while configuring your Alertmanager. If you enter an invalid Alertmanager configuration, an error message displays, and you can choose from a previous working configuration to restart it.
+
+## InfluxDB provisioning change
+
+### Database field deprecation
```

Comment:
@itsmylife - we want to avoid having two headings in a row. Can we simplify this like so:
```suggestion
## InfluxDB database field deprecation
```